ladok_generate_certificate_verify_user.sh

```diff
@@ -73,6 +73,11 @@ fi
 
 if ! choose_ladok_environment "${certificate_name}" "${password_input}"; then
     printf "\tERROR no ladok environment was found\n"
+    if openssl x509 -in "${certificate_name}".crt -passin pass:"${password_input}" -subject -noout | grep Prod-UFH; then
```

Comment:
So "uppfoljningsdatabasen" has it's own certificate? But does it need to be "Prod-UFH" or can it be any other environment?